include/nuttx/wireless/bluetooth/bt_gatt.h

```diff
@@ -40,6 +40,7 @@
  * Included Files
  ****************************************************************************/
 
+#include <sys/param.h>
```

Comment:
Could you please point out where `nitems` are used in this file as I have some troubles with identifying the exact line?

Reply:
https://github.com/apache/nuttx/pull/9087/files/2ab1ddd1c6137dda2accba69c50a7c4a72be79cb#diff-351b4670c48478a0115b1e13f504ff17b565e713ab9b906732d593bbbb0a68b7R332